### PR TITLE
cmake30: 3.0.2 - new formula

### PR DESCRIPTION
**Pull request by Tuxity**
Adding cmake 3.0 versions too

### CODE REVIEW

File: cmake30.rb [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+    EOS
+  end
+end
+
+class Cmake30 < Formula
+  homepage "http://www.cmake.org/"
+  url "http://www.cmake.org/files/v3.0/cmake-3.0.2.tar.gz"
+  sha256 "6b4ea61eadbbd9bec0ccb383c29d1f4496eacc121ef7acf37c7a24777805693e"
+
+  head do
```
Comment (DomT4) [inline cmake30.rb]: Same thing here, kill the head block.

Comment (Tuxity) [inline cmake30.rb]: Should keep the `depends_on "xz" # For LZMA` and put it outside the head block ?

Comment (DomT4) [inline cmake30.rb]: Nope, the stable version doesn't make use of ` xz ` here so you can just murder the dep :).

**Comment by Tuxity** [comment]
corrected and squashed :)

**Comment by Tuxity** [comment]
see #719

**Pull request closed without merging**